### PR TITLE
CEO-11: Add event notification

### LINKED ISSUE #11: CEO-11: Add Event notification
### Summary

Add Event notification

### Solution

- [x] Model
- [x] Service
- [x] Controller

### PR DESCRIPTION
**Pull request by Koroliuk**
## Proposed changes

Event notifications functionality was provided in scope of the MR

## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

Check following steps to help us land your changes.
_Put an `x` in the boxes of completed items._

- [X] I have read the [CONTRIBUTING](https://github.com/InvictoProjects/cultural-events-organizer/blob/main/CONTRIBUTING.md) doc
- [X] Tests and linter show no problems
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have added the necessary documentation (if appropriate)
- [X] I linked Pull Request to a corresponding Issue and added it's number to Pull Request name

### CODE REVIEW

**Comment by Pienskoi** [comment]
You deleted Gradle Wrapper and forgot to add a new one with updated version

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.1-bin.zip
-networkTimeout=10000
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-bin.zipnetworkTimeout=10000
```
Comment (Pienskoi) [inline gradle/wrapper/gradle-wrapper.properties]:
```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-bin.zip
networkTimeout=10000
```

Comment (Koroliuk) [reply gradle/wrapper/gradle-wrapper.properties]: Found this typo as well, already fixed

**Review by Pienskoi — APPROVED** [review]
SUPER ⭐

**Review by VladyslavHolik — APPROVED** [review]

**Pull request merged**